### DISCUSSION TITLE
Airdrop ne kadar kazandıracak... Kazanmak için neler yapabiliriz...

### DISCUSSION THREAD

**Discussion by Stendardo17** [IBAX-io/go-ibax#771] [Ideas]
Bir merak konusu daha. Acaba bu airdropa katılan tüm üyelerin kazancı ne kadar olacak? 
Ne kadar olmalı? Kazandıracak püf noktalar neler?

Lütfen herkes birbirine destek olsun.  Airdropun bize ne kadar kazandıracağını, ne kadar olması gerektiği hakkında FİKİRLERİNİZİ tavsiyelerinizi paylaşalım.

**Comment by Stendardo17** [comment]
İlk yorum ve tavsiye benden:
En azından kişi başına en az 25$ verilecek
Tavsiyem şu ki aynı ip üzerinden daha fazla kazanayım derken elinde olandan da olma.

**Reply by dervishh** [reply]
ben uzun vadede avax ile yarışabileceğini düşünüyorum. borsalarda listelenmesi halinde traderların ilgisini çekecektir.

**Reply by oze1** [reply]
ayni pc uzerinden birden fazla hesapla katilmak sorun olusturabilir

**Reply by Stendardo17** [reply]
Kesinlikle aynı pc, ip adresi üzerinden birden fazla hesap açarak sonradan üzülmeyelim. Ayrıca retweet yaparken 3 arkadaşınızı da ekleyin..

**Reply by Stendardo17** [reply]
100 $ olabilme ihtimali yüksek ama en düşük fiyat olarak 25$

**Reply by oze1** [reply]
ayni ip uzerinden katilanlar elenecek mi bu konuda bi aciklama var mi ?

**Reply by oze1** [reply]
cunku 1 ailede 1 bilgisayar olsa aile fertlerinin hepsi ayni pc yi kullanip katilim yapabilir

**Reply by Stendardo17** [reply]
Açıklama yok tabiki de ama her ihtimali göz önünde bulundurmak gerekir. Bunu göz ardı etmeyin.

**Reply by oze1** [reply]
testnete nasil katilabiliyoruz ve github ibax user community taskleri nerede ?

**Reply by Stendardo17** [reply]
Bu konu hakkında size yardımcı olabilmek için taslak metni oluşturacağım. Orada istediğiniz her sorunun cevabını eksiksiz olarak bulabileceksiniz. lütfen beklemede kalınız. Teşekkürler

**Comment by Azizatli21** [comment]
Bence 100 dolar olur

**Reply by Stendardo17** [reply]
Ayrıca şunu belirtmek istiyorum IBXC ler dağıtımı okurken elinizde tutmanızda fayda var. Çünkü şu an bir çok kripto en düşük günlerini yaşıyor

**Comment by oze1** [comment]
> Bir merak konusu daha. Acaba bu airdropa katılan tüm üyelerin kazancı ne kadar olacak? Ne kadar olmalı? Kazandıracak püf noktalar neler?
> 
> Lütfen herkes birbirine destek olsun. Airdropun bize ne kadar kazandıracağını, ne kadar olması gerektiği hakkında FİKİRLERİNİZİ tavsiyelerinizi paylaşalım.

testnete nasil katilim yapiyoruz
